sourcecode/hub/routes/web.php

```diff
@@ -82,6 +82,9 @@
     Route::middleware('auth:web')->group(function () {
         Route::get('/preferences', 'preferences')->name('user.preferences');
         Route::post('/preferences', 'savePreferences')->name('user.save-preferences');
+
+        Route::get('/myAccount', 'myAccount')->name('user.myAccount');
```

Comment:
The paths and names have to follow existing naming conventions. These should be `/my-account`, `user.my-account`, and `/save-name', `user.save-name`, respectively.